### PR TITLE
Update issue templates for new docs site

### PR DESCRIPTION
**Pull request by dhaavi**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/suggest-feature.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: "💡 Suggest an Improvement or Feature"
+about: Suggest an enhancement or a new feature for the Portmaster
+labels: feature
+---
+
+**What would you like to add or change?**:
+
+
+
+**Why do you and others need this?**:
```
Comment (davegson) [inline .github/ISSUE_TEMPLATE/suggest-feature.md:11]: 👌 - the only thing I would change here is add two additional newlines at the end of the file

File: .github/ISSUE_TEMPLATE/report-bug.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+**Debug Information**:
+
+<!--
+Paste debug information below:
+- General issue: Click on "Copy Debug Information" on the Settings page.
+- App related issue: Click on "Copy Debug Information" in the dropdown menu of an app in the Monitor view.
+
+⚠ Please check the "Unexpected Logs" and "Network Connections" sections in the preview mode for sensitive or otherwise private information and remove them!
```
Comment (davegson) [inline .github/ISSUE_TEMPLATE/report-bug.md:37]: I'd switch this sentence around:

⚠ Please remove sensitive/private information from the "Unexpected Logs" and "Network Connections" sections. This is easiest to do in the preview mode.

Another thing we should check is whether GitHub sends these logs to their servers before a users has the chance to remove sensitive stuff.

File: .github/ISSUE_TEMPLATE/report-bug.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+<!--
+Please disclose security related issues privately to support@safing.io.
+-->
+
+**Pre-Submit Checklist**:
+
+- Check known issues:
+  - [Windows](https://docs.safing.io/portmaster/install/windows#known-issues)
+  - [Linux](https://docs.safing.io/portmaster/install/linux#compatibility)
+- Search for similar/existing issues on Github.
```
Comment (davegson) [inline .github/ISSUE_TEMPLATE/report-bug.md:16]: I am unsure if we should move away from having checklists to fill out - ppl seemed to do this well until now. What is your take?

File: .github/ISSUE_TEMPLATE/report-compatibility.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+**Pre-Submit Checklist**:
+
+- Check applicable sources for existing issues
+  - [Linux Compatibility](https://docs.safing.io/portmaster/install/linux#compatibility)
+  - [VPN Compatibility](https://docs.safing.io/portmaster/install/status/vpn-compatibility)
+  - [Github Issues (all our repos)](https://github.com/issues?q=user%3Asafing)
+
+**Which Linux Distro, VPN Client or General Software is this about?**:
```
Comment (davegson) [inline .github/ISSUE_TEMPLATE/report-compatibility.md:14]: we must make this easier / clearer what kind of report they are creating. Maybe add all sections and make it easy to remove the ones they are not using. Or have them all in comments and then they just uncomment the one they need?

File: .github/ISSUE_TEMPLATE/report-compatibility.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+about: Report Portmaster in/compatibility with Linux Distros, VPN Clients or general Software
+labels: compatibility
+---
+
+**Pre-Submit Checklist**:
+
+- Check applicable sources for existing issues
+  - [Linux Compatibility](https://docs.safing.io/portmaster/install/linux#compatibility)
+  - [VPN Compatibility](https://docs.safing.io/portmaster/install/status/vpn-compatibility)
+  - [Github Issues (all our repos)](https://github.com/issues?q=user%3Asafing)
```
Comment (davegson) [inline .github/ISSUE_TEMPLATE/report-compatibility.md:12]: this is a stretch and surely not applicable - we will collect these only in the Portmaster repo, no?

**Review by davegson — CHANGES_REQUESTED** [review]
a few easy changes, a few things to discuss. Best have a chat about it

File: .github/ISSUE_TEMPLATE/report-compatibility.md (hunk 4 above)
Comment (dhaavi) [reply .github/ISSUE_TEMPLATE/report-compatibility.md:14]: Going with `What in/compatibility are you reporting?`.

File: .github/ISSUE_TEMPLATE/report-compatibility.md (hunk 5 above)
Comment (dhaavi) [reply .github/ISSUE_TEMPLATE/report-compatibility.md:12]: Going to go with the current repo, filtered by the respective issue tag.

**Review by davegson — DISMISSED** [review]

**Pull request merged**